clients/fides-js/src/lib/tcf.ts

```diff
@@ -107,22 +107,25 @@ export const generateFidesString = async ({
           const { id } = decodeVendorId(vendorId);
           tcModel.vendorConsents.set(+id);
 
-          // look up each vendor in the GVL vendors list to see if they have a purpose list.
-          // If they do not it means they have been set in Admin UI as Vendor Overrides to
-          // require consent. In that case we need to set a publisher restriction for the
-          // vendor's flexible purposes.
-          const vendor = experience.gvl?.vendors[id];
-          if (vendor && !vendor?.purposes?.length) {
-            vendor.flexiblePurposes.forEach((purpose) => {
-              const purposeRestriction = new PurposeRestriction();
-              purposeRestriction.purposeId = purpose;
-              purposeRestriction.restrictionType =
-                RestrictionType.REQUIRE_CONSENT;
-              tcModel.publisherRestrictions.add(+id, purposeRestriction);
-            });
+          if (!experience.tcf_publisher_restrictions?.length) {
+            // Legacy: look up each vendor in the GVL vendors list to see if they have a purpose list.
```

Comment:
Can you tell me more about when we're expecting experiences to have `tcf_publisher_restrictions` defined? Is this going to be a migration or does this require manual Admin-UI work to populate this field?

Reply:
This requires manual Admin-UI work. There needs to be a configuration added to the TCF experience and the only way to create a configuration if they have legacy in place, is to first remove the legacy. I have a loom I can send you explaining that process.

Reply:
Oh yes! If you could send the loom, that would be great, especially as I do testing tomorrow